recipe/meta.yaml

```diff
@@ -66,159 +83,147 @@ outputs:
+
+  - name: {{ name }}-notorch
     requirements:
       host:
-        - python >=3.8
+        - python
       run:
-        - python >=3.8
-        - {{ pin_subpackage("u8darts", exact=True) }}
+        - python
+        - {{ pin_subpackage("darts", exact=True) }}
+        # Dependencies from https://github.com/unit8co/darts/blob/0.26.0/requirements/notorch.txt
         - catboost >=1.0.6
         - lightgbm >=3.2.0
         - prophet >=1.1.1
-
     test:
       imports:
+        # dummy test; actual tests are in the `darts` subpackage
         - darts
-        - darts.ad
-        - darts.ad.aggregators
-        - darts.ad.anomaly_model
-        - darts.ad.detectors
-        - darts.ad.scorers
-        - darts.dataprocessing
-        - darts.dataprocessing.dtw
-        - darts.dataprocessing.encoders
-        - darts.dataprocessing.transformers
-        - darts.datasets
-        - darts.explainability
-        - darts.metrics
-        - darts.models
-        - darts.tests
-        - darts.tests.dataprocessing
-        - darts.tests.dataprocessing.transformers
-        - darts.tests.utils
-        - darts.utils
-        - darts.utils.data
-        - darts.utils.historical_forecasts
-      source_files:
-        - darts
-      commands:
-        - pytest darts/tests
-      requires:
-        - pip
-        - testfixtures
-        - pytest
 
-  - name: u8darts-torch
-    build:
-      noarch: python
+  - name: {{ name }}-torch
     requirements:
       host:
-        - python >=3.8
+        - python
       run:
-        - python >=3.8
-        - {{ pin_subpackage("u8darts", exact=True) }}
+        - python
+        - {{ pin_subpackage("darts", exact=True) }}
+        # Dependencies from torch.txt, see https://github.com/unit8co/darts/blob/0.26.0/requirements/torch.txt
         - pytorch-lightning >=1.5.0
         - tensorboardx >=2.1
         - pytorch >=1.8.0
-
     test:
       imports:
+        # dummy test; actual tests are in the `darts` subpackage
         - darts
-        - darts.ad
-        - darts.ad.aggregators
-        - darts.ad.anomaly_model
-        - darts.ad.detectors
-        - darts.ad.scorers
-        - darts.dataprocessing
-        - darts.dataprocessing.dtw
-        - darts.dataprocessing.encoders
-        - darts.dataprocessing.transformers
-        - darts.datasets
-        - darts.explainability
-        - darts.metrics
-        - darts.models
-        - darts.tests
-        - darts.tests.dataprocessing
-        - darts.tests.dataprocessing.transformers
-        - darts.tests.utils
-        - darts.utils
-        - darts.utils.data
-        - darts.utils.historical_forecasts
-      source_files:
-        - darts
-      commands:
-        - pytest darts/tests
-      requires:
-        - pip
-        - testfixtures
-        - pytest
 
-  - name: u8darts-all
-    build:
-      noarch: python
+  - name: {{ name }}-all
```

Comment:
I would keep the same package names as on conda-forge.

Reply:
IMO, the conda-forge naming of the feedstock is partially incorrect. The existence of two PyPi projects darts/u8darts makes this naming issue more difficult. darts is more popular. I would say that u8darts is deprecated

Reply:
We make darts and its synonym u8darts, and darts-all and u8darts-all subpackages

Reply:
Removed `darts`, we use `u8darts` and `u8darts-*`.

See comment https://github.com/AnacondaRecipes/u8darts-feedstock/pull/1#issuecomment-1774813629